adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -15,9 +15,43 @@
  * limitations under the License.
  */
 
-(function( window, undefined ) {
-
-    var Keycloak = function (config) {
+(function(root, factory) {
+    if ( typeof exports === 'object' ) {
+        if ( typeof module === 'object' ) {
+            module.exports = factory( require("js-sha256"), require("base64-js") );    
+        } else {
+            exports["keycloak"] = factory( require("js-sha256"), require("base64-js") );    
+        }
+    } else {
+        /**
```

Comment:
Hi,

it seems that these comments that are committed within this PR into the keycloak.js



leads to some issues with the minified version:

> Source map error: SyntaxError: JSON.parse: unexpected character at line 2 column 1 of the JSON data
Resource URL: https://DOMAIN/auth/js/keycloak.min.js
Source Map URL: keycloak.min.js.map

I don't know it is worth to create an issue, just want to mention it.

Reply:
@sassko If you can create a reproducible case of this please report the bug in the [issue tracker](https://issues.jboss.org/projects/KEYCLOAK/issues).